src/app/public/modules/image/image.component.ts

```diff
@@ -27,4 +27,19 @@ export class SkyImageComponent {
   @Input()
   public showBorder = false;
 
+  @Input()
+  public set showCaptionPrefix(value: boolean) {
+    this._showCaptionPrefix = value;
+  }
+
+  public get showCaptionPrefix(): boolean {
+    if (this._showCaptionPrefix === undefined) {
```

Comment:
It looks like this should always return true if undefined. Should we have a unit test for that?

Reply:
Technically, this is already getting tested since all of the other unit tests are setting this to `undefined` by default. See: https://github.com/blackbaud/skyux-lib-media/blob/caption-prefix/src/app/public/modules/image/fixtures/image.component.fixture.ts#L21